### ISSUE TITLE
XPP3 Support

### ISSUE THREAD

**Issue by danielpeintner** [EXIficient/exificient#4]
@Flowdalic requested in issue https://github.com/EXIficient/exificient/issues/1 support for XML Pull Parser 3rd Edition (XPP3).

**Comment by danielpeintner** [comment]
I started to play with XmlPull API and do have already some initial running code (at the moment some minor tests, see https://github.com/EXIficient/exificient/tree/master/src/test/java/com/siemens/ct/exi/api/xmlpull).

Having said that, I wonder whether there is a way to put a dependency just on the XmlPull API and not on an actual implementation. For testing purposes I added the following to Maven.

```
<dependency>
      <groupId>xpp3</groupId>
      <artifactId>xpp3</artifactId>
      <version>1.1.4c</version>
      <scope>test</scope>
</dependency>
```

Doing it without `<scope>test</scope>` puts a dependency on the XPP3 XML library which I do not want. I just want a dependency to the API (just like I did for StAX).

 @Flowdalic any idea how I can do that. Is there a public maven resource for the API only?
